testing/test_pytest_html.py

```diff
@@ -1005,12 +1005,31 @@ def test_css(self, testdir, recwarn, colors):
             assert str(v["path"]) in html
             assert v["style"] in html
 
-    def test_css_invalid(self, testdir, recwarn):
+    @pytest.mark.parametrize(
+        "file1,file2,expected",
+        [
+            ("style.css", "", "Missing CSS files: style.css,"),
+            ("abc.css", "xyz.css", "Missing CSS files: abc.css, xyz.css"),
+        ],
+    )
+    def test_css_invalid(self, testdir, recwarn, file1, file2, expected):
         testdir.makepyfile("def test_pass(): pass")
-        result = testdir.runpytest("--html", "report.html", "--css", "style.css")
+        result = testdir.runpytest(
+            "--html", "report.html", "--css", file1, "--css", file2
+        )
+        assert result.ret
+        assert len(recwarn) == 0
+        assert expected in result.stderr.str()
+
+    def test_some_css_files(self, testdir, recwarn):
```

Comment:
You can merge this test with the test above, just create the valid css file in the body of the test.